tools/common/Application.cs

```diff
@@ -38,6 +38,12 @@ public enum LinkMode {
 
 	public partial class Application
 	{
+#if MTOUCH
```

Comment:
This is a total hack, but I didn't see a better way without changing the text of an already released error.

Reply:
Since it's a partial class, can't this be included in the specific (iOS and Mac) code ?